models/model.js

```diff
@@ -17,3 +17,18 @@ exports.selectAuthorById = async (article_id) => {
     return result.rows[0];
   }
 };
+
+exports.updateArticlesById = async (article_id, newVotes) => {
+  const result = await db.query(
+    `UPDATE articles 
+   SET votes = votes + $1 
+    WHERE article_id = $2 
+    RETURNING *;`,
+    [newVotes, article_id]
+  );
+  if (!result) {
+    return Promise.reject({ status: 400, msg: "bad request" });
```

Comment:
This isn't where you handle a 400, this is a psql error. Your 404 should be handled here, and it should be if(!result.rows[0]) that you check for.